stream-platform/src/test/java/io/camunda/zeebe/stream/impl/StreamProcessorTest.java

```diff
@@ -208,6 +209,25 @@ public void shouldUpdateLastProcessPositionEvenWhenProcessingFails() {
         .until(() -> streamPlatform.getLastSuccessfulProcessedRecordPosition(), pos -> pos >= 1);
   }
 
+  @RegressionTest("not updated when instance was banned")
```

Comment:
:upside_down_face: `RegressionTest` [wants a GitHub issue link](https://github.com/camunda/zeebe/blob/8726552b5eab731aa58f8fcfa5f4af56dd0b64b0/test-util/src/main/java/io/camunda/zeebe/test/util/junit/RegressionTest.java#L18-L20). I'd suggest that we link to the comment where you first described the problem. If we don't link anywhere, the `RegressionTest` marker is useless IMO, the text describes the same thing as the method name.